lib/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@
 ##-*****************************************************************************
 
 # Include the source dirs to make the library root dir an include
-INCLUDE_DIRECTORIES(${ALEMBIC_LIBS_SOURCE_DIR})
+INCLUDE_DIRECTORIES(${ALEMBIC_LIBS_SOURCE_DIR} ${PROJECT_BINARY_DIR}/lib)
```

Comment:
I think this line should probably be pulled, as ALEMBIC_LIBS_SOURCE_DIR doesn't seem to be defined or used anywhere else.

Instead ${PROJECT_BINARY_DIR}/lib should be added to the INCLUDE_DIRECTORIES in lib/AbcOpenGL/CMakeLists.txt to better match lib/Alembic/CMakeLists.txt